alcs-api/src/user/user.service.ts

```diff
@@ -48,4 +51,24 @@ export class UserService {
       },
     });
   }
+
+  async getUserByUuid(uuid: string) {
```

Comment:
can remove the word User and just have getByUuid since this is the UserService

Reply:
I'm being consistent with the rest of the service. Do you want the rest to be renamed too?

Reply:
Ah oops, I just noticed update was called update. Should update be updateUser to be consistent then?

Reply:
I think we need to agree on the naming in general, and I can address all the API level code to be consistent across the app

Reply:
I like the idea that our service|entity|controller names include domain names in their naming and that way, we can avoid including the domain names into service method names to improve readability. If you agree, I can address the rest of API code to match it.

Reply:
As discussed this will be done in another MR with its own ticket to track.